masterlist.yaml

```diff
@@ -19576,6 +19576,36 @@ plugins:
       - crc: 0xEC8CCF97
         util: 'SSEEdit v4.0.3g'
 
+  - name: 'OpulentThievesGuild(Patch)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/931/'
+        name: 'Opulent Thieves Guild'
+  - name: 'OpulentThievesGuild.esp'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Unofficial Skyrim Special Edition Patch'
+          - '[Opulent Thieves Guild Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35261/)'
+        condition: 'active("Unofficial Skyrim Special Edition Patch.esp") and not active("Opulent Thieves Guild - USSEP Patch.esp")'
+      - <<: *wildEdits
+        subs: [ 'Remove the XEZN subrecord from RiftenRatwayZone in 00016BCF.' ]
+        condition: 'checksum("OpulentThievesGuild.esp", 78CE1FC1) and not active("OpulentThievesGuildPatch.esp")'
+    tag:
+      - Actors.ACBS
+      - Actors.AIPackages
+      - C.Light
+      - C.Music
+      - Invent.Add
+      - Invent.Remove
+      - ObjectBounds
+    clean:
+      - crc: 0xB786A56F
+        util: 'SSEEdit v4.0.4'
```

Comment:
This cleaning data should be omitted like with other dirty plugins. Looks good otherwise.

```suggestion
```